omod/src/main/java/org/openmrs/module/coreapps/fragment/controller/DeletePatientFragmentController.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.coreapps.fragment.controller;
+
+import org.openmrs.Patient;
+import org.openmrs.User;
+import org.openmrs.api.APIAuthenticationException;
+import org.openmrs.api.PatientService;
+import org.openmrs.module.appui.UiSessionContext;
+import org.openmrs.module.coreapps.CoreAppsConstants;
+import org.openmrs.ui.framework.UiUtils;
+import org.openmrs.ui.framework.annotation.SpringBean;
+import org.openmrs.ui.framework.fragment.action.FailureResult;
+import org.openmrs.ui.framework.fragment.action.FragmentActionResult;
+import org.openmrs.ui.framework.fragment.action.SuccessResult;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.web.bind.annotation.RequestParam;
+
+public class DeletePatientFragmentController {
```

Comment:
The core apps module requires the REST web services module.

-Darius (by phone)
On Apr 14, 2016 8:01 PM, "Shreyans Sheth" <notifications@github.com> wrote:

> In
> omod/src/main/java/org/openmrs/module/coreapps/fragment/controller/DeletePatientFragmentController.java
> <https://github.com/openmrs/openmrs-module-coreapps/pull/20#discussion_r59822019>
> :
>
> > +
> > +import org.openmrs.Patient;
> > +import org.openmrs.User;
> > +import org.openmrs.api.APIAuthenticationException;
> > +import org.openmrs.api.PatientService;
> > +import org.openmrs.module.appui.UiSessionContext;
> > +import org.openmrs.module.coreapps.CoreAppsConstants;
> > +import org.openmrs.ui.framework.UiUtils;
> > +import org.openmrs.ui.framework.annotation.SpringBean;
> > +import org.openmrs.ui.framework.fragment.action.FailureResult;
> > +import org.openmrs.ui.framework.fragment.action.FragmentActionResult;
> > +import org.openmrs.ui.framework.fragment.action.SuccessResult;
> > +import org.springframework.transaction.annotation.Transactional;
> > +import org.springframework.web.bind.annotation.RequestParam;
> > +
> > +public class DeletePatientFragmentController {
>
> Are you sure you want me to get rid of this entire class? In a rare
> situation, someone*just* might not have RESTWS installed and wishes to
> delete a patient then what?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/openmrs/openmrs-module-coreapps/pull/20/files/c748a5bdae230f51fdb03cd30dbab9902fa65d82#r59822019>
>